src/Temporalio/Worker/WorkflowInstance.cs

```diff
@@ -1140,13 +1145,13 @@ private void ApplyQueryWorkflow(QueryWorkflow query)
                     if (query.QueryType == "__stack_trace")
                     {
                         Func<string> getter = GetStackTrace;
-                        queryDefn = WorkflowQueryDefinition.CreateWithoutAttribute(
+                        queryDefn = WorkflowQueryDefinition.CreateWithoutAttributeReservedName(
                             "__stack_trace", getter);
                     }
                     else if (query.QueryType == "__temporal_workflow_metadata")
                     {
                         Func<Api.Sdk.V1.WorkflowMetadata> getter = GetWorkflowMetadata;
-                        queryDefn = WorkflowQueryDefinition.CreateWithoutAttribute(
+                        queryDefn = WorkflowQueryDefinition.CreateWithoutAttributeReservedName(
```

Comment:
For these two, we need to not go through interceptors to invoke them. Would be ok if they aren't even "query definitions" and just invoked directly or something (also saves you from having that bypass constructor param for query definition)